data/frame.go

```diff
@@ -43,6 +43,10 @@ type Frame struct {
 	Warnings []Warning // TODO: Remove, will be replaced with FrameMeta.Notices.
 }
 
+// Frames is a a collection of Frames as a slice of Frame pointers.
```

Comment:
Please remove redundant `a`.
`Frames is a a collection of Frames` can be a bit confusing.
What about `Frames is a slice of Frame pointers`?